_data/mentors.yml

```diff
@@ -697,7 +697,7 @@
     ISQB-TM and ISTQB-TA certified manual and automated tester for frontend and backend teams for the last ten years; a degree in Computer Engineering; lector and author of the IT school's QA Basic and QA Pro courses; survived in an abroad company and now hold interviews there.
   image: assets/images/mentors/anastasia.jpeg
   languages: English, Ukrainian, Russian
-  availability: [ 5 ]
+  availability: [ 6 ]
```

Comment:
They are active only in June? what about the next months?

Reply:
We don't have their Calendly so I can't make out. 
Once we close registrations for June, I am planning to send the new Availability form to all ad-hoc mentors so hopefully we shud get everyone's Calendly link.